packages/headless/src/api/search/search/result.ts

```diff
@@ -19,11 +20,11 @@ export type Result = {
   isTopResult: boolean;
   isRecommendation: boolean;
   isUserActionView: boolean;
-  titleHighlights: string[];
-  firstSentencesHighlights: string[];
-  excerptHighlights: string[];
-  printableUriHighlights: string[];
-  summaryHighlights: string[];
+  titleHighlights: HighlightKeyword[];
+  firstSentencesHighlights: HighlightKeyword[];
+  excerptHighlights: HighlightKeyword[];
+  printableUriHighlights: HighlightKeyword[];
+  summaryHighlights: HighlightKeyword[];
```

Comment:
Should I just change what is needed in this PR (`excerptHighlights`) or leave it with everything changed?

Reply:
Change it for all of them in order to be consistent